src/main/java/uk/gov/hmcts/ccd/domain/service/getcasedocument/CaseDocumentTimestampService.java

```diff
@@ -28,7 +30,10 @@ public void addUploadTimestamps(CaseDetails caseDetailsModified, CaseDetails cas
     protected List<String> findUrlsNotInOriginal(CaseDetails caseDetailsModified, CaseDetails caseDetailsInDb) {
         List<String> documentUrlsFromDb = getDocumentUrls(caseDetailsInDb);
 
-        List<JsonNode> jsonNodes = findNodes(caseDetailsModified.getData().values());
+        List<JsonNode> jsonNodes = new ArrayList<>();
+        if (null != caseDetailsModified && null != caseDetailsModified.getData()) {
```

Comment:
feel that the if statement would be more readable the other way around for example caseDetailsModified != null is more readable than null != caseDetailsModified but this is just preference

Reply:
I agree but, particularly for equality/assignage, null can never be assigned an alternative value. Probably a hang over from C coding standards.